backend/btrixcloud/pages.py

```diff
@@ -113,65 +129,24 @@ async def add_crawl_pages_to_db_from_wacz(self, crawl_id: str, batch_size=100):
 
             # Add any remaining pages in buffer to db
             if pages_buffer:
-                await self._add_pages_to_db(crawl_id, pages_buffer)
+                try:
+                    await self._add_pages_to_db(crawl_id, pages_buffer, ordered=False)
+                # pylint: disable=broad-exception-caught
+                except Exception as e:
+                    print("Error inserting, probably dupe", e)
```

Comment:
Same as above. Is there a specific exception for duplicate records we can catch, or results of the query we can inspect?

Reply:
If entire batches of pages are regularly dupes, there might be something off...